contact form added on  finish the mobile version

Robison-faike123 opened the pull request:
- [ ] Add a contact form to my portfolio with my  coding partner.
- [ ] We have made it that we guide our users' experience in a way that helps them fill in the form without frustration

christianbiring1 reviewed (changes requested):
Hi @Robison-faike123 , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

# Invalid Code Review Request :heavy_multiplication_x: 
![Invalid](https://user-images.githubusercontent.com/97100091/194105041-cc43376a-43a2-4bdf-affe-f545c4040db2.gif)

You have submitted a project, that has no linters set up. Kindly set the linters config and from there you will be good to go as they help our code to respect the standards

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

mckent05 commented on index.html:
```diff
@@ -237,5 +234,77 @@ <h1>Skills</h1>
+    </div>
+    <form action="https://formspree.io/f/{form_id}" method="post">
+      <div class="ContactForm">
+          <div class="inputText">
+                  <input type="text" name="name" placeholder="Full name"
+                   required="" maxlength="30" type="text" class="input_t">
+                   <div class="line-"></div>
+            </div>
```
- Some parts of your contact form does not exactly match the given Figma template. Remember it is important that you try to replicate the Figma design as closely as possible to give confidence to recruiters in your ability to produce exact business requirements 🙏. To set the bottom border of your input I'd suggest you use the `border-bottom` property in CSS.
- For semantic reasons and to improve your site's accessibility, the `required` attribute does not need to be assigned a value. Please fix this issue 👍 👍

Your project(Pay attention to the form input):
<img width="283" alt="figma template2" src="https://user-images.githubusercontent.com/73607512/194250547-6d1d3c84-d5ea-4c3d-9285-305041854519.PNG">

Figma template:
<img width="144" alt="original figma contact" src="https://user-images.githubusercontent.com/73607512/194251237-4c4f9b8e-6e87-47bb-af79-4229d18b97f7.PNG">

mckent05 commented on index.html:
```diff
@@ -237,5 +234,77 @@ <h1>Skills</h1>
+          I'm always interested in hearing about new projects, so if you'd like
+          to chat please get in touch.
+        </p>
+    </div>
+    <form action="https://formspree.io/f/{form_id}" method="post">
```
- I get an error when I try to submit form details on your site. This is because you have not used a valid form_id. I'd suggest you signup on and use your given id [formspree](https://formspree.io/html). Please kindly fix this 👍 👍

mckent05 commented on index.html:
```diff
@@ -237,5 +234,77 @@ <h1>Skills</h1>
+                  <input type="Submit" value="Get In Touch" class="btn-touch">
+            </div>
+        </div>
+    </form>
```
- For improved semantics and site accessibility, I'd suggest you don't assign a value to the `required` attribute. Please kindly fix this issue 🙏

mckent05 reviewed (changes requested):
Hi @Robison-faike123 , 

# STATUS: CHANGES REQUIRED ♻️ ♻️ 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
- Linter checks are passing correctly ✔️ ✔️ 
- Your repo looks professional 👏 👏 
- Your PR title and PR summary are descriptive 👍 👍 


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
**OPTIONAL** - I noticed some of your images in your website are not displaying, especially your project cards. This could be because you are navigating your images folder wrongly. I'd suggesting you try `./images/example.png` when navigating into your image dir. Please kindly fix this issue for all images as it may apply. 😄 

**OPTIONAL** - Also I'd suggest you make your site responsive, this will help your site fit correctly across all screen sizes. Please fix this issue 👍 👍 

Your project:
<img width="268" alt="project card" src="https://user-images.githubusercontent.com/73607512/194257869-fabb6a12-f2ec-43c5-8a39-eea83d4f9b52.PNG">

Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Robison-faike123 commented:
- [ ] all the recommendations and suggestions have be put into consideration .but however, the linters you recommended to set was already on and I deleted it then I replaced it back .for that am requesting for shortage procedure to do so incase it happen again I will solve it without waiting much time

ichala commented on index.html:
```diff
@@ -237,5 +234,77 @@ <h1>Skills</h1>
+          to chat please get in touch.
+        </p>
+    </div>
+    <form action="https://formspree.io/f/xyyvjvdn" method="POST">
+      <div class="ContactForm">
+          <div class="inputText">
+                  <input type="text" name="name" placeholder="Full name"
+                   required="" maxlength="30" type="text" class="input_t">
+                   <div class="line-"></div>
+            </div>
+            
+
+          <div class="inputText">
+              <input
+               type="email"name="email" required=""
+                placeholder="Email address" class="input_t">
+                  
+           </div>
+           
+
+          <div class="inputTextarea">
+              <textarea
+               id="comment"
+               name="comment"
+               cols="50" rows="10" maxlength="500" required=""
+               placeholder="Enter text here"
+               type="text" class="text-area"></textarea>
+            </div>
+            
+            <div class="contact-button">
+                  <input type="Submit" value="Get In Touch" class="btn-touch">
+            </div>
+        </div>
+    </form>
```
- [ ] good job adding form id and fixing the required attribute, but as mentioned by the previous reviewer kindly try to follow the Figma design as much as possible, try to follow these steps :

1. kindly reduce the space between the input and the line below it 
2. try to change the `textarea` background color to match the Figma design, the correct color is `#FBF8F7` and add some padding inside
3. kindly try to center the social icons in the middle of the page like shown in figma design


![image](https://user-images.githubusercontent.com/89282221/194296217-84d193eb-497e-4112-8068-6147f58ef20f.png)

Robison-faike123 replied:
the color in text area to me fills more useful and interactive to the user than the one in figma and that's why they say "we can use the figma colour" but its not a must to use them

Robison-faike123 replied:
Now this is the last time to review this project but to my understanding and to what you recommended it would have been sorted already.3/3 review on this project coming so lets be considerate to the students.

ichala reviewed (changes requested):
Hi @Robison-faike123 , 

Good job so far! 👏👏👏 
There are some issues that you still need to work on to go to the next project but you're almost there!

![the-simpsons-homer-simpson](https://user-images.githubusercontent.com/28307960/157369993-50d5ba12-099f-4f2a-9a19-5e57ee2d1c96.gif)

## To Highlight 🎉

- formspree id added✔️ 
- `required` attribute fixed ✔️ 

 ## Required Changes ♻️

- [ ] Check the comments below 


## Optional suggestions

- N/A


**********
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Robison-faike123 commented:
the things you consider to be major in real sense its optional and only call for suggestion but can't be considered major. In fact the little as it is can cause someone to hurt even the all process. I think the recommendation "reduce the padding in input before merging " would sound more encouraging and supportive than blocking the approval.

Santiago220991 commented on styles.css:
```diff
@@ -705,3 +733,168 @@ button:hover {
+  color: #5b5957;
+}
+
+.footer {
+  display: flex;
+  justify-content: space-between;
+  margin: 20px;
+  padding: 15px 14px;
+  border-top: #dbd8d7;
+}
+
```
Kindly make sure you are using the same color for the social media icons provided by the Figma template as [the last code reviewer suggested](https://github.com/Robison-faike123/mobile-me-/pull/3#discussion_r988892950). :eyes: :eyes: 

Your design:
![image](https://user-images.githubusercontent.com/98363075/194327034-a86b6b4e-d993-401b-aa3b-300552d1cb8f.png)

Figma design:
![image](https://user-images.githubusercontent.com/98363075/194327146-5eafc994-4843-44d3-a3e7-a9d568684623.png)

- [OPTIONAL] Please make sure that your project sticks as much as possible to the Figma template provided. Kindly add the line below the social media icons. :pray: :pray:

Santiago220991 replied:
Hi, @Robison-faike123 I made a mistake I already informed Microverse. The last code reviewer never asked to change the color of the icons. I'm sorry your project should be approved.

Robison-faike123 replied:
you said its optional then why didn't you approve then I work on the rest as we go on with studies.

Lameck1 replied:
@Robison-faike123 Please don't ask for another code review. We are working on resolving this.

Robison-faike123 replied:
https://github.com/Robison-faike123/mobile-me-/pull/3

that is the same pull request .please kindly find time and review it
